### PR TITLE
Adds Vue Formulate to relevant lists

### PR DESCRIPTION
**Pull request by andrew-boyd**
We're proud to have launched Vue Formulate 2.0 as an open-source form authoring solution for Vue.js. This PR adds Vue Formulate to 3 lists which all apply to the capabilities of the tool.

Additions have been made to the bottom of the lists on:
- Generator
- Form
- Form -> Validation

### CODE REVIEW

**Comment by andrew-boyd** [comment]
@Scrum Hello! Definitely still relevant. I've updated to resolve conflicts.

File: README.md [hunk 1]
```diff
@@ -2207,6 +2209,7 @@ composing CSS breakpoint state.
  - [vue-final-validate](https://phphe.github.io/vue-final-validate/) - Vue validation solution from my development experience, support nested, async.
  - [vform](https://github.com/cretueusebiu/vform) - A simple way to handle Laravel back-end validation in Vue.
  - [vue-form-container](https://github.com/VitorLuizC/vue-form-container) - A `Provider Component` that encapsulate your forms and handle their states and validations.
+ - [Vue Formulate](https://github.com/wearebraid/vue-formulate) - Vue form authoring solution which includes comprehensive front-end validation functionality and a simple API for custom front-end validation rules.
```
Comment (Scrum) [inline README.md]: @andrew-boyd duplicate

File: README.md [hunk 2]
```diff
@@ -2187,6 +2188,7 @@ composing CSS breakpoint state.
  - [vue-form-components](https://github.com/antonreshetov/vue-form-components) - Clean & minimal vue form elements with validation
  - [ncform](https://github.com/ncform/ncform) - a very nice configuration generation way to develop form
  - [v-ajax-form](https://github.com/hosokawat/v-ajax-form) -  vue.js component plug-in to make Form tag compatible with ajax request
+ - [Vue Formulate](https://github.com/wearebraid/vue-formulate) - The easiest way to build forms with Vue.
```
Comment (Scrum) [inline README.md]: @andrew-boyd duplicate

**Comment by Scrum** [comment]
@andrew-boyd ping ?

**Comment by andrew-boyd** [comment]
@Scrum I added [Vue Formulate](https://vueformulate.com) in three places because it applies to each of those three lists. Vue Formulate is a form [authoring](https://vueformulate.com/guide/inputs/) tool, a form [validation](https://vueformulate.com/guide/validation/) tool, and a form [generating](https://vueformulate.com/guide/forms/#generating-forms) tool all-in-one. Is it not kosher to have it listed within each section that It applies to?

**Comment by Scrum** [comment]
@andrew-boyd No, I think that the tool has a mainstream and everything else is its options

**Comment by andrew-boyd** [comment]
Ok, I’ll need to re-look at the readme and see where it best applies if there only one entry allowed. I’ll update the PR today, thanks!

**Pull request merged**